tests/test_tti.py

```diff
@@ -87,41 +87,3 @@ def test_tti(shape, space_order, kernel):
 
     log("Difference between acoustic and TTI with all coefficients to 0 %2.4e" % res)
     assert np.isclose(res, 0.0, atol=1e-4)
-
-
-@pytest.mark.parametrize('shape', [(50, 60), (50, 60, 70)])
-def test_tti_staggered(shape):
```

Comment:
why is this being completely dropped ?

Reply:
This test was oiriginally there to check the loop are split (which the new test does).
This is like suuuuper expensive to run 3D tti with zero compiler optimization so removed it now that the one thing tested here is tested in test_dse